webview/src/setup/components/remote/MultiProjectRemotes.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react'
+import { RemoteList } from 'dvc/src/setup/webview/contract'
+import styles from './styles.module.scss'
+
+export const MultiProjectRemotes: React.FC<{
+  remoteList: NonNullable<RemoteList>
+}> = ({ remoteList }) => (
+  <table data-testid="remote-details" className={styles.remoteDetails}>
+    <tbody>
+      {Object.entries(remoteList).map(([dvcRoot, remotes]) => {
+        return remotes ? (
+          Object.entries(remotes).map(([alias, remote], i) => (
+            <tr key={[alias, remote].join('-')}>
+              {i === 0 ? <td className={styles.alias}>{dvcRoot}</td> : <td />}
+              <td className={styles.alias}>{alias}</td>
```

Comment:
Should this be a `th`?

Reply:
Unrelated to this pr, but we also should be using `th` in the dvc env details table. I'll open a quick pr later correcting that :)